src-java/kilda-model/src/main/java/org/openkilda/model/IpSocket.java

```diff
@@ -0,0 +1,26 @@
+/* Copyright 2021 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.model;
+
+import lombok.NonNull;
+import lombok.Value;
+
+@Value
+public class IpSocket {
```

Comment:
The name is confusing and clashes with java.net.Socket which is "class implements client sockets". I'd propose "IpSocketAddress"

Reply:
Ok. Renaming.